apps/ui/src/Placeholder.tsx

```diff
@@ -1,16 +1,31 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-export interface PlaceholderProps {
+import { cva, VariantProps } from 'class-variance-authority';
+
+const placeholderStyle = cva(
+	'h-[1em] w-full animate-shimmer rounded-[3px] bg-placeholderShimmer bg-[length:1000px_100%]',
+	{
+		variants: {
+			rounded: {
+				'3px': 'rounded-[3px]',
```

Comment:
nit: I'd call this `md` or something to give it more semantic meaning than `3px`

Reply:
updated